squeal-postgresql/src/Squeal/PostgreSQL/Type/List.hs

```diff
@@ -107,3 +108,25 @@ Length '[Char,String,Bool,Double] :: Nat
 type family Length (xs :: [k]) :: Nat where
   Length '[] = 0
   Length (_ : xs) = 1 + Length xs
+
+{- | `Sublist` checks that one type level list is a sublist of another,
```

Comment:
The examples display the ordering, but perhaps you could be explicit that this checks if the first list is a sublist of the second.